metafix/src/test/java/org/metafacture/metafix/MetafixScriptTest.java

```diff
@@ -430,41 +429,41 @@ private void assertStrictness(final Metafix.Strictness strictness, final boolean
     }
 
     @Test
-    @MetafixToDo("See https://github.com/metafacture/metafacture-fix/pull/170")
     public void shouldSkipExpressionOnExecutionException() {
         assertStrictness(Metafix.Strictness.EXPRESSION, true, o -> {
             o.get().startRecord("1");
-            o.get().literal("before", "");
             o.get().literal("data", "FOO");
+            o.get().literal("before", "");
```

Comment:
Then I think it would be better to call the fields "one", "two" (not "before", "after").

Reply:
But sure, call it whatever makes (more) sense to you.